docs/design/Abstract Syntax Tree.md

```diff
@@ -1,3 +1,5 @@
+# This Doc covers the ExprNode AST, which has been deprecated and removed from the Codebase. 
+
```

Comment:
I think we need to see if I want to remove this file or change this file so it introduces PIG AST.

Reply:
Probably worth another PR to clean up the documentation in the code base. Issue is created: https://github.com/partiql/partiql-lang-kotlin/issues/879